src/utility.cpp

```diff
@@ -89,8 +90,23 @@ namespace azure {  namespace storage_lite {
 #else
         pm = std::gmtime(&t);
 #endif
-        size_t s = std::strftime(buf, 30, (format == date_format::iso_8601 ? constants::date_format_iso_8601 : constants::date_format_rfc_1123), pm);
-        return std::string(buf, s);
+        if (format == date_format::iso_8601)
+        {
+            char buf[32];
+            std::strftime(buf, sizeof(buf), constants::date_format_iso_8601, pm);
```

Comment:
Why iso_8601 does not need locale adjustment?

Reply:
talked offline

Reply:
because iso_8601 is locale-independent